my_enums.rb

```diff
@@ -115,4 +260,7 @@ def multiply_els
   end
```

Comment:
* [ ] my_inject combines all elements of enum by applying a binary operation, specified by a block
  ```ruby
    range = Range.new(5, 50)
    p range.my_inject(4) { |prod, n| prod * n } # returns this error
    #NoMethodError:
    #    undefined method `[]' for 5..50:Range
  ```

* [ ] my_inject when a symbol is specified combines each element of the collection by applying the symbol as a named method
  ```ruby
    range = Range.new(5, 50)
    p range.my_inject(2, :*) # returns this error
    #NoMethodError:
    #    undefined method `[]' for 5..50:Range
  ```